pkg/arvo/lib/hood/drum.hoon

```diff
@@ -80,24 +79,21 @@
   ::
   =-  (turn - |=(a=term home+a))
   ^-  (list term)
-  ?:  lit
-    :~  %dojo
-        %eth-watcher
-        %azimuth-tracker
-    ==
   %+  welp
-    ?:  ?=(%pawn (clan:title our))  ~
-    :~  %acme
-        %dns
+    :~  %dojo
+        %spider
         %eth-watcher
         %azimuth-tracker
+        %ping
     ==
+  ?:  lit
+    ~
   :~  %lens
+      %clock
       %dojo
       %modulo
```

Comment:
%modulo app has been removed because its functionality was very minor and made sense to just fold it into %launch.

Reply:
thanks, removed